include/ginkgo/core/base/executor.hpp

```diff
@@ -511,6 +511,31 @@ class Executor : public log::EnableLogging<Executor> {
         this->run(op);
     }
 
+    /**
+     * Runs one of the passed in functors, depending on the Executor type
+     * with a custom operation name.
+     *
+     * @tparam ClosureOmp  type of op_omp
+     * @tparam ClosureCuda  type of op_cuda
+     * @tparam ClosureHip  type of op_hip
```

Comment:
missing dpcpp

```suggestion
     * @tparam ClosureHip  type of op_hip
     * @tparam ClosureDpcpp  type of op_dpcpp
```

Also in lines 497 and 502